index.html

```diff
@@ -78,9 +89,9 @@ <h3 class="storysubtitle">Multi-Post Stories</h3>
                     <h2 class="projecttitle">Professional Art Priting Data</h2>
                     <p class="projectparagraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
                     <ul class="skill">
-                        <li><a href=""><img class="skill-image-tag" src="./images/project/Tag0.png" alt=""></a></li>
-                        <li><a href=""><img class="skill-image-tag" src="./images/project/Tag1.png" alt=""></a></li>
-                        <li><a href=""><img class="skill-image-tag" src="./images/project/Tag2.png" alt=""></a></li>
+                        <li><a href=""><img class="skill-image-tag" src="./images/project/Tag0.png" alt="Rubi legend, as reference of the back end language used on the project"></a></li>
+                        <li><a href=""><img class="skill-image-tag" src="./images/project/Tag1.png" alt="Bootstrap legend, as reference of the tool used for styling the page"></a></li>
+                        <li><a href=""><img class="skill-image-tag" src="./images/project/Tag2.png" alt="HTML legend, as reference of the coding used on the front end"></a></li>
                     </ul>
                     <button type="submit" class="projectbutton">See Project</button>
```

Comment:
Kindly change the contrast to be at least `4.5` of the `see project` buttons because they failed in the contract check
![image](https://user-images.githubusercontent.com/24703171/155246262-000d53f1-5da8-4d97-9105-f806bde8032b.png)
You can check the contrast for your website using this [link](https://www.aditus.io/button-contrast-checker/)